Automattic/wp-calypso
# Social | Add Threads to social connections UI

## Proposed Changes

* Update Threads UIs for social connections

## Why are these changes being made?


* We are adding support for [threads.net](https://threads.net) in Social

## Testing Instructions



* Sandbox D152810-code and follow the instructions given there
* Point `public-api.wordpress.com` to your sandbox
* Goto `/marketing/connections/:site`
* Confirm that you see Threads in the connections list
* Confirm that you see "New" badge on it
* Confirm that you don't see "New" badge on Nextdoor
* Confirm that the descriptions are corrected both in collapsed and expanded form
* Connect a Threads account
* Confirm that it works as expected
* Confirm the same for Jetpack Cloud

## Pre-merge Checklist




<img width="893" alt="image" src="https://github.com/Automattic/wp-calypso/assets/18226415/2564b172-f5c5-4cfb-bb4b-2bbfd092e7a0">


<img width="730" alt="Screenshot 2024-06-21 at 1 28 07 PM" src="https://github.com/Automattic/wp-calypso/assets/18226415/b8749fd7-798a-42dd-a5a9-885bba2b54c3">
